nengo/networks/workingmemory.py

```diff
@@ -5,15 +5,13 @@
 
 
 def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
-                     difference_gain=1.0, reset_gain=3,
-                     mem_config=None, net=None):
+                     difference_gain=1.0, reset_gain=3, net=None):
     """Stores a given vector in memory, with input controlled by a gate."""
     if net is None:
         net = nengo.Network(label="Input Gated Memory")
 
-    if mem_config is None:
-        mem_config = nengo.Config(nengo.Connection)
-        mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
+    mem_config = nengo.Config(nengo.Connection)
+    mem_config[nengo.Connection].synapse = nengo.Lowpass(0.1)
```

Comment:
I'm +1 on removing ```reset_gain``` and ```gate_gain```.  I'm also +1 on removing ```difference_synapse```, but I'm willing to let @xchoo 's vote count stronger than mine. 

Also, speaking of parameters, ```fdbk_scale``` should be changed to be just ``feedback``` (or whatever it's being called in the equivalent case in ```spa.State```), for consistency.

As for discussing at the next Nengo-dev meeting, I think most of the main interested parties are already involved, so hopefully we can sort it out here...  although we might want to move it to the main discussion thread rather than in this comment on an outdated commit....  :)